sdk/ml/azure-ai-ml/azure/ai/ml/_schema/_distillation/distillation_job.py

```diff
@@ -0,0 +1,87 @@
+from azure.ai.ml._schema.job.input_output_fields_provider import OutputsField
+from azure.ai.ml._schema.job_resource_configuration import JobResourceConfigurationSchema
+from azure.ai.ml._schema.workspace.connections import ServerlessConnectionSchema, WorkspaceConnectionSchema
+from azure.ai.ml._utils._experimental import experimental
+from azure.ai.ml.constants import DataGenerationTaskType, DataGenerationType, JobType
+from azure.ai.ml.constants._common import AzureMLResourceType
+
+
+@experimental
+class DistillationJobSchema(FineTuningVerticalSchema, BaseJobSchema):
+    type = StringTransformedEnum(required=True, allowed_values=JobType.DISTILLATION)
+    data_generation_type = StringTransformedEnum(
+        allowed_values=[DataGenerationType.LabelGeneration, DataGenerationType.DataGeneration],
+        required=True,
+    )
+    data_generation_task_type = StringTransformedEnum(
+        allowed_values=[
+            DataGenerationTaskType.NLI,
+            DataGenerationTaskType.NLU_QA,
+            DataGenerationTaskType.CONVERSATION,
+            DataGenerationTaskType.MATH,
+            DataGenerationTaskType.SUMMARIZATION,
+        ],
+        casing_transform=str.upper,
+        required=True,
+    )
+    teacher_model_endpoint_connection = UnionField(
+        [NestedField(WorkspaceConnectionSchema), NestedField(ServerlessConnectionSchema)], required=True
+    )
+    student_model = UnionField(
+        [
+            NestedField(ModelInputSchema),
+            RegistryStr(azureml_type=AzureMLResourceType.MODEL),
+            ArmVersionedStr(azureml_type=AzureMLResourceType.MODEL, allow_default_version=True),
+        ],
+        required=True,
+    )
+    training_data = UnionField(
+        [
+            NestedField(DataInputSchema),
+            ArmVersionedStr(azureml_type=AzureMLResourceType.DATA),
+            fields.Str(metadata={"pattern": r"^(http(s)?):.*"}),
+            fields.Str(metadata={"pattern": r"^(wasb(s)?):.*"}),
+            LocalPathField(pattern=r"^file:.*"),
+            LocalPathField(
+                pattern=r"^(?!(azureml|http(s)?|wasb(s)?|file):).*",
+            ),
+        ]
+    )
+    validation_data = UnionField(
+        [
+            NestedField(DataInputSchema),
+            ArmVersionedStr(azureml_type=AzureMLResourceType.DATA),
+            fields.Str(metadata={"pattern": r"^(http(s)?):.*"}),
+            fields.Str(metadata={"pattern": r"^(wasb(s)?):.*"}),
+            LocalPathField(pattern=r"^file:.*"),
+            LocalPathField(
+                pattern=r"^(?!(azureml|http(s)?|wasb(s)?|file):).*",
+            ),
+        ]
+    )
+    teacher_model_settings = NestedField(TeacherModelSettingsSchema)
+    prompt_settings = NestedField(PromptSettingsSchema)
+    hyperparameters = fields.Dict(keys=fields.Str(), values=fields.Str(allow_none=True))
+    resources = NestedField(JobResourceConfigurationSchema)
+    outputs = OutputsField()
+
+    class Meta:  # type: ignore
```

Comment:
where is Meta getting used?

Reply:
Removed. Was used to exclude 'task' and 'model' due to having FIneTuningVertical as a parent class but removed it as a parent class so no more need for Meta